pinot-segment-local/src/main/java/org/apache/pinot/segment/local/indexsegment/mutable/MutableSegmentImpl.java

```diff
@@ -586,18 +603,19 @@ private void addNewRow(int docId, GenericRow row) {
 
           // Update forward index
           DataType dataType = fieldSpec.getDataType();
+          value = column.equals("*") ? 1 : value;
```

Comment:
For aggregated metrics, we should set the value to `ValueAggregator.getInitialAggregatedValue(value)`